docs/current/guides/snippets/use-services/environment-variables/main.py

```diff
@@ -0,0 +1,33 @@
+import sys
+
+import anyio
+import dagger
+
+def env_variables(envs):
+    def wrapper(ctr):
+        for key, value in envs.items():
+            ctr = ctr.with_env_variable(key, value)
+        return ctr
+    return wrapper
+
+async def main():
+    config = dagger.Config(log_output=sys.stdout)
+
+    async with dagger.Connection(config) as client:
+        # Container setup
```

Comment:
Some minor changes suggested to comment strings, should replicate in each language.